modules/internal/Bytes.chpl

```diff
@@ -1112,6 +1112,20 @@ module Bytes {
     return createBytesWithNewBuffer(x.buff, length=x.numBytes, size=x.numBytes+1);
   }
 
+  // Cast from c_string to bytes
```

Comment:
We've been talking about deprecating cast from c_string to string... can the new factor functions be used instead?